modules/configuring-vsphere-connection-settings.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-vSphere-connection-settings_{context}"]
+= Configuring the vSphere connection settings
+
+[role="_abstract"]
+Modify the following vSphere configuration settings:
+
+* vCenter network address
+* vCenter username
+* vCenter password
+* vCenter address
+* vSphere data center
+* vSphere datastore
+* Virtual machine of the cluster
+
+.Prerequisites
+* The {ai-full} has finished installing the cluster successfully.
+* The cluster is connected to link:https://console.redhat.com/.
```

Comment:
Doc shows `The cluster is connected to link:https://console.redhat.com/.` maybe we should remove the word `link:`